module/VuFindSearch/src/VuFindSearch/Backend/Primo/Response/RecordCollection.php

```diff
@@ -97,6 +97,9 @@ public function getOffset()
     /**
      * Return any errors.
      *
+     * Each error can be a translatable string or an array with keys 'message' and
```

Comment:
This isn't a Blender-specific change, is it? Should we cherry pick this bit live since it's just a comment improvement?

Reply:
Not just a comment improvement, there's actual functionality using it. Previously it was just a string.